Content.Server/Botany/Components/PlantAnalyzerComponent.cs

```diff
@@ -0,0 +1,40 @@
+using Robust.Shared.Audio;
+
+namespace Content.Server.Botany.Components;
+
+/// <summary>
+///    After scanning, retrieves the target Uid to use with its related UI.
+/// </summary>
+[RegisterComponent]
+public sealed partial class PlantAnalyzerComponent : Component
+{
+    /// <summary>
+    ///     How long it takes to scan.
+    /// </summary>
+    [DataField("scanDelay")]
```

Comment:
```suggestion
    [DataField]
```
Datafield names can be autogenerated now. Don't specify it unless it's different from the C# field name.